app/src/main/res/layout/activity_explorer.xml

```diff
@@ -30,9 +30,9 @@
 
             <ListView
                 android:id="@+id/list_explorer"
+                style="@style/listViewNoDivider"
                 android:layout_width="match_parent"
-                android:layout_height="match_parent"
-                style="@style/listViewNoDivider"/>
+                android:layout_height="match_parent" />
```

Comment:
Do you have anything against my properties ordering 😂 ?

Reply:
Nah, it's my IDE. It just keeps reformatting every file I open :sweat_smile: